kube/sample-app.yaml

```diff
@@ -0,0 +1,97 @@
+# Sample app backend
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sample-app-backend
+  namespace: sample-app
+spec:
+  selector:
+    matchLabels:
+      app: sample-app-backend
+  template:
+    metadata:
+      labels:
+        app: sample-app-backend
+    spec:
+      volumes:
+        - name: host-mount
+          hostPath:
+            path: '/src/sample-app'
+      containers:
+        - name: sample-app-backend
+          imagePullPolicy: IfNotPresent
+          image: sample-app-backend:latest
+          workingDir: '/src/sample-app/backend'
+          volumeMounts:
```

Comment:
Maybe add a comment `# Disable this when deploying to non-dev environments` or such?